paparazzi/src/test/java/app/cash/paparazzi/internal/resources/PseudolocaleGeneratorTest.kt

```diff
@@ -0,0 +1,209 @@
+/*
+ * Copyright (C) 2023 Square, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package app.cash.paparazzi.internal.resources
+
+import com.android.ide.common.rendering.api.ArrayResourceValue
+import com.android.ide.common.rendering.api.PluralsResourceValue
+import com.android.ide.common.rendering.api.ResourceNamespace
+import com.android.ide.common.rendering.api.ResourceValue
+import com.android.ide.common.resources.KnownNamespacesMap
+import com.android.ide.common.resources.ResourceValueMap
+import com.android.ide.common.resources.configuration.LocaleQualifier
+import com.android.resources.ResourceType
+import com.google.common.collect.Table
+import com.google.common.collect.Tables
+import com.google.common.truth.Truth.assertThat
+import org.junit.Test
+import java.util.EnumMap
+
+class PseudolocaleGeneratorTest {
+  private val resFolderRoot = resolveProjectPath("src/test/resources/folders/res")
+
+  private val repository = ResourceFolderRepository(
+    resFolderRoot.toFile(),
+    ResourceNamespace.TODO()
+  )
+
+  private fun createResourceTable(): Table<ResourceNamespace, ResourceType, ResourceValueMap> {
+    val map: Map<ResourceNamespace, Map<ResourceType, ResourceValueMap>> = KnownNamespacesMap()
+    val table = Tables.newCustomTable(map) { EnumMap(ResourceType::class.java) }
+    repository.allResources.groupBy { it.type }.forEach { (type, items) ->
+      table.put(
+        ResourceNamespace.TODO(),
+        type,
+        ResourceValueMap.create().apply {
+          putAll(items.map { it.name to it as ResourceValue })
+        }
+      )
+    }
+
+    return table
+  }
+
+  @Test
+  fun testAccentPseudolocalization() {
+    val result = createResourceTable()
+    result.pseudolocalizeIfNeeded(LocaleQualifier.getQualifier("en-rXA"))
+
+    // STRING
+    with(result[ResourceNamespace.TODO(), ResourceType.STRING]!!) {
+      assertThat(size).isEqualTo(4)
+      assertThat(get("string_name")?.value).isEqualTo("[Ţéšţ Šţŕîñĝ one two]")
```

Comment:
let's either migrate from `get` to `getValue`, or use the `!!`.  null-safe'ing defers the crash to the assertion instead of throwing NPE right away.

same with the other instances below.